packages/synapse-react-client/package.json

```diff
@@ -122,15 +122,15 @@
     "@jest/reporters": "^29.5.0",
     "@juggle/resize-observer": "^3.3.1",
     "@rollup/plugin-inject": "^4.0.4",
-    "@storybook/addon-actions": "^7.0.6",
-    "@storybook/addon-essentials": "^7.0.6",
-    "@storybook/addon-interactions": "^7.0.6",
-    "@storybook/addon-links": "^7.0.6",
-    "@storybook/manager-api": "^7.0.6",
-    "@storybook/react": "^7.0.6",
-    "@storybook/react-vite": "^7.0.6",
+    "@storybook/addon-actions": "^7.0.9",
```

Comment:
Was having some local issues with Storybook so I upgraded to the latest version to try to fix them.